### DISCUSSION TITLE
Use GitHub Releases

### DISCUSSION THREAD

**Discussion by quinncomendant** [ghostty-org/ghostty#3826] [Feature Requests, Ideas] [answered]
Suggestion: A new GitHub Release should be created for each new Ghostty version. The release should include full changelog notes with links to PR/commits. This would be helpful because it allows me to subscribe to release announcements in a way consistent with other GitHub projects: the release announcement with changelog is sent to my email address and appears in GitHub notifications.

<img width="444" alt="image" src="https://github.com/user-attachments/assets/2965fb04-f342-4a29-8fe1-ed56948a96d6" />

---

I'm soooo excited for Ghostty! Thanks for your vision and making this reality!

**Comment by JakeWharton** [comment]
Not sure what the change log plan is. There was discussion (on Discord maybe?) about whether to do entries into a file as part of the PR, as part of the commit (e.g., `Relnote:` stanza), or something else.

Once that is determined, it'll be trivial to add automatically creating the GitHub release into the tag workflow.

**Reply by mitchellh** [reply, marked as answer]
I agree, let's move this discussion to the changelog discussion. It was started prior to 1.0 and never closed so it should exist.

**Reply by JakeWharton** [reply]
#3071

**Comment by reitzig** [comment]
FWIW, creating releases (on top of the existing tags) is useful _even if_ there is no (good) changelog, because at least we get that notification that there's something new.

**Reply by rhodes-b** [reply]
like stated above this has moved into #3071
